src/main/java/pl/nogacz/checkers/board/Board.java

```diff
@@ -35,7 +46,7 @@
 
     private boolean isGameEnd = false;
     private int roundWithoutKick = 0;
-
+    public boolean isMenuActive=false;

```

Comment:
Do not make this public. Instead implement an accessor method.

Reply:
Done